pantheon-systems/drops-8
# Update to Drupal 8.2.2.

For more information, see https://www.drupal.org/project/drupal/releases/8.2.2

Verification instructions:

- Confirm that the [Circle tests](https://circleci.com/gh/pantheon-systems/drops-8/31?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) passed (and actually ran meaningful tests)
- Inspect the [installed site](http://c8-31-ci-drops-8.pantheonsite.io/). (Get a login link from the [admin dashboard](https://admin.dashboard.pantheon.io/sites/689219ca-6583-4af8-ab05-2cebf6ef79a0#c8-31/code) if necessary)

Instructions for creating your own test site:

- Create a new Drupal 8 site on Pantheon.
- When site creation is finished, visit dashboard.
- Switch to "git" mode.
- Clone your site locally.
- Apply the files from this PR on top of your local checkout.
  - git remote add drops-8 git@github.com:pantheon-systems/drops-8.git
  - git fetch drops-8
  - git merge drops-8/update-8.2.2
- Push your files back up to Pantheon.
- Switch back to sftp mode.
- Visit your site. Step through the installation process. Look for any problems.
